plugins/pencilblue/controllers/actions/salesforce/logout.js

```diff
@@ -27,6 +27,8 @@ module.exports = function LogOutSFSSOControllerModule(pb) {
                         if (err) {
                             reject(false);
                         } else {
+                            //close the session (destroys all, including session uid)
+                            pb.session.close(this.session, () => {});
```

Comment:
I get your changes, but you agree that if line 23 fails, this wont be executed right? If that's the intended behavior, I don't see any problem with this code.

Reply:
Yes, the idea is to remove the dependency with SF. We want to delete the session now, regardless of what SF does @sindydanely